src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +20,14 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(() -> new
+                DataProcessingException("Movie with id: "
+                + id
+                + " not found", null));
```

Comment:
The `get` method uses `orElseThrow` to throw a `DataProcessingException` with a `null` cause. Consider providing a more meaningful exception message or handling the absence of a movie more gracefully.